__tests__/configuration_string.test.js

```diff
@@ -0,0 +1,16 @@
+import TestUtils from './test-utils';
+
+describe('configuration string', () => {
+  it('should return 400 without a token parameter', async () => {
+    const response = await TestUtils.request
+      .get('/configuration-string');
+    expect(response.status).toBe(400);
+  });
+
+  it('should return 200 with token in parameter', async () => {
```

Comment:
Should we have test cases for when the token is passed but is invalid, and for when it's passed and valid?

In other words, tests for the cases when `success` it True and False in the response.

Reply:
Testing this in the mock test is not that good, that's why I try to focus my tests on the integration tests.

The success tests is already implemented in the create token and create nft test files. I've added a new test file to run the invalid token uid test: https://github.com/HathorNetwork/hathor-wallet-headless/pull/225/commits/0b9b19d305314ae0cf9ee8f0185a67ada6e942e1